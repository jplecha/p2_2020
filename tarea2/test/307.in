insertarAlFinal (9,3.5)
insertarAlFinal (5,1.9)
insertarAlFinal (5,6.1)
insertarAlFinal (8,2.8)
insertarAlFinal (7,4.9)
insertarAlFinal (3,1.4)
insertarAlFinal (4,9.7)
insertarAlFinal (6,3.7)
insertarAlFinal (5,8.4)
localizadorEnCadena
finalCadena
imprimirCadena
esVaciaCadena
finalCadena
insertarAlFinal (5,6.3)
imprimirCadena
finalCadena
insertarSegmentoDespues (4,2.9) (9,1.9)
borrarSegmento 2 1
localizadorEnCadena
finalCadena
finalCadena
finalCadena
imprimirCadena
