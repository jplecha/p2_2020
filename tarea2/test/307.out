==12754== Conditional jump or move depends on uninitialised value(s)
==12754==    at 0x4012F4: leerCadena() (principal.cpp:79)
==12754==    by 0x401B06: main (principal.cpp:237)
==12754== 
principal: principal.cpp:250: int main(): Assertion `esVaciaCadena(cad) || ((1 <= k1) && (k1 <= k2) && (k2 <= longitud(cad)))' failed.
1>Insertado al final.
2>Insertado al final.
3>Insertado al final.
4>Insertado al final.
5>Insertado al final.
6>Insertado al final.
7>Insertado al final.
8>Insertado al final.
9>Insertado al final.
10>loc no pertenece a cad.
11>loc al final.
12>(9,3.50)(5,1.90)(5,6.10)(8,2.80)(7,4.90)(3,1.40)(4,9.70)(6,3.70)(5,8.40)
13>cad no vacia.
14>loc al final.
15>Insertado al final.
16>(9,3.50)(5,1.90)(5,6.10)(8,2.80)(7,4.90)(3,1.40)(4,9.70)(6,3.70)(5,8.40)(5,6.30)
17>loc al final.
18>Segmento insertado despuÃ©s de loc.
19>timeout: the monitored command dumped core
